File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/ghost/TriggerValues.java
package org.firstinspires.ftc.teamcode.util.ghost;

import java.util.ArrayList;
import java.util.Arrays;

public class TriggerValues extends ControllerValues<Double> {
    public final static String leftTrigger = "lt";
    public final static String rightTrigger = "rt";

    public TriggerValues() {
        super(0.0,
                new ArrayList<>(Arrays.asList(new Double[]{0.0, 0.0})),
                new ArrayList<>(Arrays.asList(new String[]{leftTrigger, rightTrigger}))
        );
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/ghost/TriggerValues.kt
package org.firstinspires.ftc.teamcode.util.ghost


class TriggerValues : ControllerValues<Double>(
        0.0,
        arrayListOf(0.0, 0.0),
        arrayListOf(leftTrigger, rightTrigger)
) {
    companion object {
        const val leftTrigger = "lt"
        const val rightTrigger = "rt"
    }

}
